igor-web/src/main/groovy/com/netflix/spinnaker/igor/gitlabci/service/GitlabCiService.java

```diff
@@ -94,37 +112,117 @@ public Permissions getPermissions() {
   }
 
   public List<Project> getProjects() {
-    return getProjectsRec(new ArrayList<>(), 1);
+    return getProjectsRec(new ArrayList<>(), 1, 100).parallelStream()
```

Comment:
Same question about the 100 here.

Reply:
100 is the largest page size you can request for Gitlab, so defaulting there makes the most sense for performance given the high number of API calls to get all the info Spinnaker expects from Gitlab.  I made this configurable as well (default still 100).